env0/resource_template_test.go

```diff
@@ -308,8 +308,12 @@ func TestUnitTemplateResource(t *testing.T) {
 		if template.TokenId != "" {
 			templateAsDictionary["token_id"] = template.TokenId
 		}
-		if template.GitlabProjectId != 0 {
-			templateAsDictionary["gitlab_project_id"] = template.GitlabProjectId
+		if template.Id == gitlabTemplate.Id {
+			if template.Name == gitlabUpdatedTemplate.Name {
+				templateAsDictionary["gitlab_project_id"] = gitlabTemplateUpdatedProjectId
```

Comment:
This is where I set the gitlab_project_id... the "if" is used to detect the relevant use-case